src/layouts/hass-subpage.ts

```diff
@@ -34,10 +45,18 @@ class HassSubpage extends LitElement {
         <div main-title>${this.header}</div>
         <slot name="toolbar-icon"></slot>
       </div>
-      <div class="content"><slot></slot></div>
+      <div class="content" @scroll=${this._saveScrollPos}><slot></slot></div>
     `;
   }
 
+  private _saveScrollPos: EventListenerOrEventListenerObject &
```

Comment:
Think there is a decorator for setting passive. But why not just store on disconnect?

Reply:
Tried that, but was not available of disconnect